drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/src/main/java/org/drools/workbench/screens/scenariosimulation/webapp/client/dropdown/ScenarioSimulationKogitoRuntimeCreationAssetsDropdownProviderImpl.java

```diff
@@ -64,6 +66,8 @@ public void getItems(Consumer<List<KieAssetsDropdownItem>> assetListConsumer) {
     }
 
     protected KieAssetsDropdownItem getKieAssetsDropdownItem(final String asset) {
-        return new KieAssetsDropdownItem(asset, "", asset, new HashMap<>());
+        int idx = asset.replaceAll("\\\\", "/").lastIndexOf('/');
```

Comment:
What is the reason for this? Can you add a simple test for this class?

Reply:
The scope is to extract the filename from a fullPath string.
I'll provide tests.